Steps4Impact/Settings/SettingsViewController.swift

```diff
@@ -53,8 +53,15 @@ class SettingsViewController: TableViewController {
     case .viewTeam:
       navigationController?.pushViewController(TeamSettingsViewController(), animated: true)
     case .leaveTeam:
-      // TODO(compnerd) handle this
-      break
+      let alert: AlertViewController = AlertViewController()
+      alert.title = Strings.TeamSettings.leave
+      alert.body = Strings.TeamSettings.leaveBody
+      alert.add(AlertAction(title: "Cancel", style: .secondary, shouldDismiss: true))
+      alert.add(AlertAction(title: "Leave", style: .destructive, shouldDismiss: true) {
```

Comment:
weak self on closures to avoid retain cycle

Reply:
good call!